### ISSUE TITLE
[BUG] Wrong readme example for authnContext for ADFS

### ISSUE THREAD

**Issue by jyenduri-uptycs** [node-saml/passport-saml#642]
** Spec-driven development **

This project is focused on compliance with the SAML 2.0 specification. For any bug report that
involves the SAML spec, please link to the related parts of the spec and quote the passages too.

Start here: http://saml.xml.org/saml-specifications

You might also check the spec to confirm that it doesn't address your particular bug and mention
that you found no references in the spec concerning your issue.

** Community development model **

passport-saml is maintained by a number of current users. There is no author or primary maintainer
waiting to write your tests and documentation for you. To increase the odds that your issue
is promptly dealt with, consider a pull request to address the issue that includes test coverage
and updated documentation.

**To Reproduce**

I am trying out ADFS  based on the example `authContext` is a string, but looks like it is only working if we give it as an array

{
    entryPoint: 'https://ad.example.net/adfs/ls/',
    issuer: 'https://your-app.example.net/login/callback',
    callbackUrl: 'https://your-app.example.net/login/callback',
    cert: 'MIICizCCAfQCCQCY8tKaMc0BMjANBgkqh ... W==',
    authnContext: 'http://schemas.microsoft.com/ws/2008/06/identity/authenticationmethod/windows',
    identifierFormat: null
  }

Working example

{
    entryPoint: 'https://ad.example.net/adfs/ls/',
    issuer: 'https://your-app.example.net/login/callback',
    callbackUrl: 'https://your-app.example.net/login/callback',
    cert: 'MIICizCCAfQCCQCY8tKaMc0BMjANBgkqh ... W==',
    authnContext: ['http://schemas.microsoft.com/ws/2008/06/identity/authenticationmethod/windows'],
    identifierFormat: null
  }

**Expected behavior**
We need to update the README and examples

**Environment**

- Node.js version: v12.14.0
- passport-saml version: 3.1.2

**Comment by jyenduri-uptycs** [comment]
@cjbarth  Can you please look at this pull request https://github.com/node-saml/passport-saml/pull/647

**Comment by ctaschereau** [comment]
Agreed. This appears to be an undocumented breaking change with the version 2.x since we had working code passing only a string as an authncontext before...

**Comment by cjbarth** [comment]
I found the commit that created the break. It was my [commit](https://github.com/node-saml/passport-saml/pull/548), and I overlooked updating the documentation. Thanks for catching this.
